src/cforth/decomp2.fth

```diff
@@ -45,7 +44,9 @@ d# 300 2* /n* constant /positions
 
 headers
 defer indent
-: (indent)  ( -- )  lmargin @ #out @ - 0 max spaces  ;
+: (indent)  ( -- )  lmargin @ #out @ -
+\ 0 max          \ unnecessary, as spaces does that too.
+   spaces  ;
```

Comment:
Formatting:  if the definition fits entirely on one line, it can follow the stack diagram.  Otherwise the entire body must be on a 3-space indent and the ; is on a line by itself